docs/data/base/components/tabs/tabs.md

```diff
@@ -9,79 +9,79 @@ packageName: '@mui/base'
 
 {{"demo": "UnstyledTabsCustomized.js"}}
 
 ## Third-party routing library
 
-One frequent use case is to perform navigation on the client only, without an HTTP round-trip to the server.
-The `Tab` component provides the `component` prop to handle this use case.
-Here is a [more detailed guide](/material-ui/guides/routing/#tabs).
+A common use case for tabs is to implement client-side navigation that doesn't require an HTTP round-trip to the server.
+
+The `TabUnstyled` component provides the `component` prop to handle this use case—see [the Material UI documentation on routing](/material-ui/guides/routing/#tabs) for more details.
 
 ## Accessibility
 
 (WAI-ARIA: https://www.w3.org/TR/wai-aria-practices/#tabpanel)
 
-The following steps are needed in order to provide necessary information for assistive technologies:
+The following steps are necessary to make the tab component suite accessible to assistive technology:
 
-1. Label `TabsUnstyled` via `aria-label` or `aria-labelledby`.
-2. `TabUnstyled`s need to be connected to their corresponding tab panels by setting the correct `id`, `aria-controls` and `aria-labelledby`.
+1. Label `TabsUnstyled` with `aria-label` or `aria-labelledby`.
+2. Connect each `TabUnstyled` component to its corresponding `TabPanelUnstyled` by setting the correct `id`, `aria-controls` and `aria-labelledby`.
 
-An example for the current implementation can be found in the demos on this page.
+The demos below illustrate the proper use of ARIA labels.
 
 ### Keyboard navigation
 
-The components implement keyboard navigation using the "manual activation" behavior.
-If you want to switch to the "selection automatically follows focus" behavior you have pass `selectionFollowsFocus` to the `Tabs` component.
-The WAI-ARIA authoring practices have a detailed guide on [how to decide when to make selection automatically follow focus](https://www.w3.org/TR/wai-aria-practices/#kbd_selection_follows_focus).
+By default, when using keyboard navigation, the tab components open via **manual activation**—that is, the next panel is displayed only after the user activates the tab with either <kbd class="key">Space</kbd>, <kbd class="key">Enter</kbd>, or a mouse click.
+
+This is the preferable behavior for tabs in most cases, acccording to [the WAI-ARIA authoring practices](https://www.w3.org/TR/wai-aria-practices/#kbd_selection_follows_focus).
 
-#### Demo
+Alternatively, you can set the panels to be displayed automatically when their corresponding tabs are in focus—this behavior of the selection following the focus is known as **automatic activation**.
 
-The following two demos only differ in their keyboard navigation behavior.
-Focus a tab and navigate with arrow keys to notice the difference, e.g. <kbd class="key">Arrow Right</kbd>.
+To enable automatic activation, pass the `selectionFollowsFocus` prop to the `TabsUnstyled` component:
```

Comment:
Now reading this section, maybe we should rename the prop from `selectionFollowsFocus` to `automaticActivation` :)